lib/gooddata/models/project_creator.rb

```diff
@@ -204,6 +210,52 @@ def pick_correct_chunks(chunks, opts = {})
           end
         end
 
+        def pick_correct_chunks_hard_sync(chunks, opts = {})
+          GoodData.logger.debug("update_preference") { opts[:update_preference].pretty_inspect }
```

Comment:
we should remove GoodData.logger.debug before merging.

Reply:
Done remove it. We also don't need log them for each project.